examples/01.Basics/Sinewave_HIFI/Sinewave_HIFI.ino

```diff
@@ -41,15 +35,18 @@
     Tim Barrass 2012-13, CC by-nc-sa.
 */
 
-#include <MozziGuts.h>
+#include <MozziConfigValues.h>
+#define MOZZI_AUDIO_MODE MOZZI_OUTPUT_2PIN_PWM
```

Comment:
Clean! 
I wonder if, in the future, it might be worth to give access to some configurations with several defines, straight with a macro (I am thinking of something close to the `MIDI_CREATE_INSTANCE` from the Midi library for instance.

Reply:
You mean e.g. setting 2pin-mode along with the pins to use in one macro?